assets/javascripts/modules/analytics/ga.js

```diff
@@ -64,8 +64,19 @@ define(['modules/analytics/analyticsEnabled',
         ga('jellyfishGA.send', 'pageview');
     }
 
+    function trackEvent(eventLabel) {
+        var upgrading = (eventLabel === 'Rate Plan Change' && guardian.pageInfo.productData.initialProduct !== guardian.pageInfo.productData.productPurchasing) ? 1 : 0;
+        ga('membershipPropertyTracker.send', 'event', {
+            eventCategory: 'Subscriptions Checkout',
+            eventAction:  guardian.pageInfo.productData.productType,
+            eventLabel: eventLabel,
+            dimension11: guardian.pageInfo.productData.productType + ' - ' + guardian.pageInfo.productData.productPurchasing,
```

Comment:
We can't change the field name of the object, but I will put into a variable. See this Google doc for what each of the dimensions are: https://docs.google.com/spreadsheets/d/1MmWHNeeiQE_dzekImIP9Tv4beLx_8JzWx3rOtCp4PGg/edit